docs/variants/qemu_q35/hardware-matrix.md

```diff
@@ -0,0 +1,29 @@
+# Hardware configuration matrix
+
+## Introduction
+
+This document describes the devices emulated by the QEMU Q35.
+
+## QEMU Q35 //TBD
```

Comment:
When do you plan to add actual information about QEMU emulated hardware?

Reply:
The problem is with the internet connectivity to the Ubuntu with OVMF, I am working on it (by trying libvirt as of now), due to the known issue of optionROM.

Reply:
@maheshtammisetti OST2 Arch4021 should teach you all thing required for networking configuration in QEMU. `The problem` is not correct explanation of the error you face please be more precise when you describing problems.

Reply:
> due to the known issue of optionROM

The issue is not know unless you add it to https://github.com/Dasharo/dasharo-issues/issues with the explanation what is happening, error logs, etc. Now you are trying some libvirt method which we have no idea if it will work and leaving the original problem unsolved...

Reply:
Added Hardware matrix after the fix of OptionROM issue